sites/svelte-5-preview/src/routes/docs/content/03-appendix/01-faq.md

```diff
@@ -154,8 +154,8 @@ We appreciate your enthusiasm! We welcome issues on the [sveltejs/svelte](https:
 
 ### How can I share feedback or cool examples of what this enables?
 
-You can use the `#svelte-5-alpha` channel on the [Discord server](https://svelte.dev/chat) or the tag `#svelte-5-alpha` on social media.
+You can use the `#svelte-5-rc` channel on the [Discord server](https://svelte.dev/chat) or the tag `#svelte-5-rc` on social media.
```

Comment:
I don't know that we want to change the hashtag here, too.

Also, do we want to have a specific hashtag? Has this been a well used thing?

Reply:
I can't find any occurrences on Twitter (admittedly search there seems pretty broken nowadays). If we _were_ to keep the hashtag I'd just drop the qualifier

```suggestion
You can use the `#svelte-5-rc` channel on the [Discord server](https://svelte.dev/chat) or the tag `#svelte-5` on social media.
```